examples/examples_ts_test.go

```diff
@@ -18,6 +19,36 @@ func TestOrgWlanTs(t *testing.T) {
 		opttest.LocalProviderPath("pulumi-junipermist", filepath.Join(getCwd(t), "..", "bin")),
 		opttest.YarnLink("@pulumi/juniper-mist"),
 	)
-	test.SetConfig("organizationId", os.Getenv(EnvMistOrgID))	
+	test.SetConfig("organizationId", os.Getenv(EnvMistOrgID))
 	test.Up()
 }
+func TestOrgWlanIsolationTs(t *testing.T) {
+	// Regression test for https://github.com/pulumi/pulumi-junipermist/issues/49
+	checkBaseEnvVars(t)
+	test := pulumitest.NewPulumiTest(t, "org-wlan-isolation-ts",
+		opttest.LocalProviderPath("pulumi-junipermist", filepath.Join(getCwd(t), "..", "bin")),
+		opttest.YarnLink("@pulumi/juniper-mist"),
+	)
+	test.SetConfig("organizationId", os.Getenv(EnvMistOrgID))
+	test.Up()
+	preview, error := test.CurrentStack().Preview(test.Context())
```

Comment:
slight nit - you can use test.Preview() which will assert that no error has happened under the hood.

I believe preview also takes an optpreview.ExpectNoChanges(), which acts like the the cli --expect-no-changes

Reply:
are you saying to instead do something like 
```
	test.SetConfig("organizationId", os.Getenv(EnvMistOrgID))
	test.Up()
	assertpreview.HasNoChanges(t, test.Preview())
```